lib/src/Base/Optim/NLopt.cxx

```diff
@@ -364,7 +364,7 @@ void NLopt::run()
       Point h(getProblem().getInequalityConstraint()(inP));
       for (UnsignedInteger k = 0; k < getProblem().getInequalityConstraint().getOutputDimension(); ++ k)
       {
-        h[k] = std::max(h[k], 0.0);// convention h(x)>=0 <=> admissibility
+        h[k] = std::min(h[k], 0.0);// convention h(x)>=0 <=> admissibility
```

Comment:
If we expect a positive error, it should be:
```
h[k] = -std::min(h[k], 0.0);// convention h(x)>=0 <=> admissibility
```
or
```
h[k] = std::max(-h[k], 0.0);// convention h(x)>=0 <=> admissibility
```
no?

Reply:
we have to only take into account the violated constraints (h<0)
then we compute the inf norm of h